functional-tests/pom.xml

```diff
@@ -10,35 +10,49 @@
    <properties>
       <maven.compiler.source>1.8</maven.compiler.source>
       <maven.compiler.target>1.8</maven.compiler.target>
-
+      <version.arquillian.bom>1.1.8.Final</version.arquillian.bom>
       <infinispan-client-hotrod.version>8.5.0.DR2-redhat-1</infinispan-client-hotrod.version>
       <openshift-client.version>3.0.3</openshift-client.version>
+      <infinispan-remote.version>8.4.0.Final-redhat-2</infinispan-remote.version>
+      <openshift-client.version>2.6.1</openshift-client.version>
       <arquillian-junit-standalone.version>1.1.13.Final</arquillian-junit-standalone.version>
       <arquillian-cube-openshift.version>1.9.0</arquillian-cube-openshift.version>
       <assertj-core.version>3.8.0</assertj-core.version>
       <rest-assured.version>3.0.5</rest-assured.version>
+      <version.chameleon>1.0.0.Beta2</version.chameleon>
    </properties>
 
+   <dependencyManagement>
+      <dependencies>
+         <dependency>
+            <groupId>org.jboss.arquillian</groupId>
+            <artifactId>arquillian-bom</artifactId>
+            <version>${version.arquillian.bom}</version>
+            <type>pom</type>
+            <scope>import</scope>
+         </dependency>
+      </dependencies>
+   </dependencyManagement>
+
    <dependencies>
       <!-- https://github.com/arquillian/arquillian-cube/blob/master/docs/kubernetes.adoc#dealing-with-version-conflicts -->
       <dependency>
          <groupId>org.infinispan</groupId>
-         <artifactId>infinispan-client-hotrod</artifactId>
-         <version>${infinispan-client-hotrod.version}</version>
+         <artifactId>infinispan-remote</artifactId>
```

Comment:
Perhaps we could avoid uber jars here - WDYT? They are not shipped by default in the upstream, so maybe we also should use small jars.

Reply:
I don't know. To me uberjars still seem easier to use. They definitely cause us problems when building them but they're easier to use for customers. Not sure why we should be avoiding them.

Reply:
If we use the individual jars we're only pulling in what we need and it's much easier to debug if there are any dependency issues.